Fix for: https://github.com/msukmanowsky/ROmniture/issues/2

jaredhabeck opened the pull request:
Updated fix with test coverage.

Commit:
Return response body on json parse failure

msukmanowsky commented:
Thanks Jared - I actually don't have access to Omniture anymore.  Mind showing me the output of the test results you ran locally?

jaredhabeck commented:
No problem. Looks like one of your previous tests is failing due it not actually raising an exception as was expected. The test I added is passing no problem, I also reverted the code to its previous state of not rescuing the bad JSON output to verify the test failed and it did.

```
Finished tests in 16.063810s, 0.2490 tests/s, 0.3735 assertions/s.

1) Failure:
test_a_bad_request(ClientTest) [test/client_test.rb:45]:
ROmniture::Exceptions::OmnitureReportException expected but nothing was raised.

4 tests, 6 assertions, 1 failures, 0 errors, 0 skips
```
